apollo-router/src/services/layers/persisted_queries/mod.rs

```diff
@@ -216,14 +221,32 @@ impl PersistedQueryLayer {
         }
 
         match manifest_poller.action_for_freeform_graphql(operation_body, db.ast(file_id)) {
-            FreeformGraphQLAction::Allow => Ok(request),
-            FreeformGraphQLAction::Deny => Err(supergraph_err_operation_not_in_safelist(request)),
+            FreeformGraphQLAction::Allow => {
+                tracing::info!(monotonic_counter.apollo.router.operations.persisted_queries = 1u64,);
+                Ok(request)
+            }
+            FreeformGraphQLAction::Deny => {
+                tracing::info!(
+                    monotonic_counter.apollo.router.operations.persisted_queries = 1u64,
+                    persisted_queries.safelist.rejected.unknown = false,
```

Comment:
I know this is a quite old change, but I'm not sure why this attribute is false here and true in line 247.  We are rejecting it as unknown in both cases; the only difference is whether we log, which has its own attribute. I think this should be changed to true, which I guess is OK because this change describes these metrics as unstable and I don't know that we ever changed that?

Reply:
I think this is a bug, it was probably just a typo.

Reply:
Makes sense. I fixed it in #6403.